packages/react/src/components/Card/Card.module.css

```diff
@@ -1,120 +1,104 @@
-.card {
-  display: flex;
-  flex-direction: column;
-  overflow: hidden;
-  width: 330px;
+.media {
+  width: auto;
 }
 
-a.card {
-  color: inherit;
-  text-decoration: none;
-}
-
-.card > img,
-.card > picture {
+.media > * {
+  display: flex;
+  flex-direction: column;
   width: 100%;
-  height: auto;
-}
-
-.card.neutralBackground {
-  background-color: var(--fds-semantic-surface-neutral-default);
-}
-
-.card.subtleBackground {
-  background-color: var(--fds-semantic-surface-neutral-subtle);
-}
-
-.card.neutralBackground:hover,
-.card.subtleBackground:hover {
-  background-color: var(--fds-semantic-surface-neutral-subtle-hover);
+  border: 0;
 }
 
-.card.neutralBackground:active,
-.card.subtleBackground:active {
-  background-color: var(--fds-semantic-surface-neutral-subtle-hover);
-}
-
-.card.firstBackground {
-  background-color: var(--fds-semantic-surface-first-light);
-}
-
-.card.firstBackground:hover {
-  background-color: var(--fds-semantic-surface-first-light-hover);
-}
-
-.card.firstBackground:active {
-  background-color: var(--fds-semantic-surface-first-light-active);
+.footer,
+.content {
+  display: flex;
+  justify-content: flex-start;
+  gap: var(--fds-spacing-4);
+  flex-wrap: wrap;
+  word-wrap: break-word;
+  font: var(--fds-typography-paragraph-medium);
+  font-family: var(--fdsc-typography-font-family);
 }
 
-.card.secondBackground {
-  background-color: var(--fds-semantic-surface-second-light);
+.content {
+  flex-direction: column;
 }
 
-.card.secondBackground:hover {
-  background-color: var(--fds-semantic-surface-second-light-hover);
+.header {
+  display: flex;
+  flex-direction: column;
+  padding: var(--fds-spacing-2) 0;
+  flex-wrap: wrap;
+  word-wrap: break-word;
+  font: var(--fds-typography-heading-medium);
+  font-family: var(--fdsc-typography-font-family);
 }
 
-.card.secondBackground:active {
-  background-color: var(--fds-semantic-surface-second-light-active);
+.group {
+  display: flex;
+  gap: var(--fds-spacing-2);
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
 }
 
-.card.thirdBackground {
-  background-color: var(--fds-semantic-surface-third-light);
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  position: relative;
+  overflow: hidden;
```

Comment:
I am assuming this is for the `Card.Media` to not overflow, but I think it might be better to use the child position of `.media`, and set border-radius directly on the media.
Might run into issues when changing the border-radius on card, but we could probably use `inherit`?

Reply:
I started with doing as you suggested ("media > *:first/last border-radius" ), but the borders didn't look quite right as it wasn't filled out correctly in the top always. Also tried different box-sizings. So testing the overflow approach now, which seems to work the best atm. Drawnback is well, `overflow:hidden` 🙈  

I cant decide which solution I like the best so up for sparring :)

Reply:
Have you tried a wrapper around the `children` of media with `border-radius` and `hidden` on that?

Reply:
Good suggestion! I'll try that :)

Reply:
Still getting the unsynced border-radiuses with your suggestion :/ 

Using `border-radius` on `Card.Media` children:
![image](https://github.com/digdir/designsystem/assets/1070981/2a596d9c-1f19-4b49-b683-9a10c6fb8ca5)

Using `overflow:hidden` on `Card`:
![image](https://github.com/digdir/designsystem/assets/1070981/0fc685f3-5aea-4506-957f-6fc5c6f36062)

Reply:
Yeah 🙈